app/workloadSummaryReport/services/workloadSummaryActions.js

```diff
@@ -46,6 +46,20 @@ class WorkloadSummaryActions {
 						courses.list[course.id] = course;
 					});
 
+					// Adding census data to find last offering
+					courses.ids.forEach(function (courseId) {
+						var course = courses.list[courseId];
+						course.census = [];
+						var SNAPSHOT_CODE = "CURRENT";
+						DwService.getDwCensusData(course.subjectCode, course.courseNumber).then(function (courseCensus) {
```

Comment:
There should be an empty line between then statement and the variable declaration above it